network.go

```diff
@@ -113,6 +113,13 @@ func checkNetworkState() {
 	}
 }
 
+func RestartMDNS() {
+	err := startMDNS()
+	if err != nil {
+		return
```

Comment:
Do we care if the MDNS doesn't restart? Should we be checking that before calling `SaveConfig()` above? Seems like we're just eating the error here but carrying on with the save there (error or not).